src/main/kotlin/com/jetbrains/micropython/settings/MicroPythonSettingsPanel.kt

```diff
@@ -45,6 +45,7 @@ class MicroPythonSettingsPanel(private val module: Module) : JPanel() {
       update()
     }
   }
+  private val clearReplOnLaunch = CheckBox("Clear REPL window each time it starts")
```

Comment:
Why do you want to introduce this option? Isn't it better to always clear the REPL on launch?

Reply:
Not really, since sometimes you may need to look up the output of the previous command and/or compare results of the previous executions.

Reply:
So is it only about clearing the output? Will we close the REPL session and restart it after flashing files to the device independently of that? Then I get it.

Reply:
Yes, it is a cosmetic option and has no bearing on the management of REPL session.